profiles.md

```diff
@@ -1,14 +1,43 @@
 ---
 title: "The Rodents"
 ---
+<br>
+<br>
 
-
-
-### Dipodamys merriami
+### _Dipodomys merriami_ - Merriam's kangaroo rat
 One of three Kangaroo Rats found at the site. In constant competition with D. ordii
 
-### Dipodamy ordii
+### _Dipodomys ordii_ - Ord's kangaroo rat
 The other regular Kangaroo Rat
 
-### Chaetodipus penicillatus
-Commonly the most abundant rodent at Portal. 
\ No newline at end of file
+### _Dipodomys spectabilis_ - banner-tailed kangaroo rat
+The largest of the three kangaroo rats at the site, _D. spectabilis_ (fondly referred to as "spectabs" by those in the know) these majestic kangaroo rats are know for their striking tails. They have by far the largest feet on the site at a whopping 48 mm average and often weigh in at over 100 grams, which is nearly twice that of the other kangaroo rat weights. Spectabs were once running the show at Portal. As the site became shrubbier, however, the reign of _D. spectabilis_ came to a slow end in the 1990's. Since then, a few individuals have popped up here and there, but they haven't stuck around, often heading for grassier pastures.
```

Comment:
"twice that of the other kangaroo rat weights" - > "twice the size of the other kangaroo rats"